source/beamline/BioXAS/BioXASMainBeamline.h

```diff
@@ -57,12 +58,30 @@ class BioXASMainBeamline : public AMBeamline
 
 	/// Destructor.
 	virtual ~BioXASMainBeamline();
-
+    /// Returns true if all beamline components are connected, false otherwise.
```

Comment:
As a general comment, what are your tab spacings set to?  They don't seem to match your older code.